### PR TITLE
Edge: move edge config out of "common" config

### PR DESCRIPTION
**Pull request by ryantxu**
Modifies #76 so that the edge configuration is not in the "common" package.

* allows entering any region (not just from selection)
* shows a warning when the endpoint is not specified

### CODE REVIEW

File: src/common/regions.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { SelectableValue } from '@grafana/data';
+
+export const standardRegions: Array<SelectableValue<string>> = [
```
Comment (ryantxu) [inline src/common/regions.ts:3]: @bymattoa can you find the real list of valid regions?

File: package.json [hunk 2]
```diff
@@ -12,10 +12,10 @@
   "author": "Grafana Labs <team@grafana.com> (https://grafana.com)",
   "license": "Apache-2.0",
   "devDependencies": {
-    "@grafana/data": "latest",
-    "@grafana/runtime": "latest",
+    "@grafana/data": "7.3.0",
```
Comment (ryantxu) [inline package.json:15]: 7.4 is "latest" now... but we should still depend on 7.3 until we actually want to force it

**Comment by ryantxu** [comment]
@macrovve can you take a look and see if this looks reasonable?

**Comment by macrovve** [comment]
For sure, it looks good to me.

**Pull request merged**